encoding/go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/philippgille/gokv/encoding
 
 go 1.18
+
+require google.golang.org/protobuf v1.31.0
```

Comment:
With this dependency, there should now also be a `go.sum` file

Reply:
good catch, added